doc/sphinx/addendum/extraction.rst

```diff
@@ -254,12 +263,24 @@ a closed term, and of course the system cannot guess the program which
 realizes an axiom.  Therefore, it is possible to tell the system
 what ML term corresponds to a given axiom. 
 
-.. cmd:: Extract Constant @qualid => @string
+.. cmd:: Extract Constant @qualid {* @string__tv } => {| @ident | @string }
 
    Give an ML extraction for the given constant.
-   The :token:`string` may be an identifier or a quoted string.
 
-.. cmd:: Extract Inlined Constant @qualid => @string
+   :n:`@string__tv`
+     If the type scheme axiom is an arity (a sequence of products followed
+     by a sort), the then some type
+     variables have to be given (as quoted strings). The syntax is then:
```

Comment:
I personally don't understand any of this but it should clearly be rewritten, maybe with help from @coq/extraction-maintainers. 

Note the left-over "The syntax is then:" from the previous version.